test/cljam/t_common.clj

```diff
@@ -1,11 +1,35 @@
 (ns cljam.t-common
-  (:use [clojure.java.io :only [file]])
   (:require [digest]
+            [clojure.java.io :refer [file]]
             [cljam.sam :as sam]
             [cljam.bam :as bam]
             [cljam.io :as io]
             [cavia.core :as cavia :refer [defprofile with-profile]]))
 
+(defn- _in-cloverage? []
+  (let [in? (try
+              (eval '(var cloverage.coverage/*covered*))
+              true
+              (catch Throwable _ nil))]
+    ;; (prn :debug 'in-cloverage? in?)
+    in?))
+
+(def ^:private in-cloverage? (memoize _in-cloverage?))
+
+(defn- expand-deftest [sym args bodies]
```

Comment:
Thanks to your review.
I fixed these defects by https://github.com/chrovis/cljam/pull/45/commits/26e57312a55d17a8cffba6e4e1da39c2ae9fa9cc .